game/cards/dm06/angel_command.go

```diff
@@ -0,0 +1,51 @@
+package dm06
+
+import (
+	"duel-masters/game/civ"
+	"duel-masters/game/family"
+	"duel-masters/game/fx"
+	"duel-masters/game/match"
+)
+
+func GarielElementalOfSunbeams(c *match.Card) {
+
+	c.Name = "Gariel, Elemental Of Sunbeams"
+	c.Power = 7500
+	c.Civ = civ.Light
+	c.Family = []string{family.AngelCommand}
+	c.ManaCost = 5
+	c.ManaRequirement = []string{civ.Light}
+
+	spellcast := false
+
+	c.Use(fx.Creature, fx.Doublebreaker, func(card *match.Card, ctx *match.Context) {
+
+		ctx.Match.ApplyPersistentEffect(func(ctx2 *match.Context, exit func()) {
```

Comment:
I don't think we need to use a persistent effect here, should be enough to listen to SpellCast and EndOfTurn event to set spellcast to true/false. Also, a new persistent effect is being added for every event that is being fired in the match here which is probably not what was intended